README.md

```diff
@@ -49,6 +49,13 @@ settings
 
 ### Building
 
+#### Dependencies required
+
+- curl
```

Comment:
we should fix this, shouldn't depend on curl

Reply:
Apparently it does some curl to download LMDB 0.9.31, and also the test using your data uses curl also:

https://github.com/damus-io/nostrdb/blob/f8d0bbdc0144d32a56433b73f495dd65514a0326/Makefile#L104

https://github.com/damus-io/nostrdb/blob/f8d0bbdc0144d32a56433b73f495dd65514a0326/Makefile#L143

Reply:
fixed in c228caf4d520465bb24e8ca2f6d631043f6e5b66

the ndb-v0 make rule shouldn't be active anymore

Reply:
many-events is used in bench which is optional

Reply:
Noted. Fixing it !